experimental/devices/st7567/st7567.go

```diff
@@ -305,7 +305,7 @@ func New(p spi.Port, dc gpio.PinOut, rst gpio.PinOut, cs gpio.PinIn, o *Opts) (*
 		cs:  cs,
 	}
 
-	cmd := make([]byte, 9)
+	var cmd []byte
```

Comment:
Can you do instead:

```
cmd := [...]byte{
  byte(o.Bias),
  byte(o.SegmentDirection),
...
}
```

?

Reply:
Sure. With latest commit it uses the shorthand operator.